src/main/resources/application.yml

```diff
@@ -104,7 +104,9 @@ docmosis:
     welshStatementOfTruth:
       templateName: FL-PRB-SOT-WEL-00234.doc
     assembledLetter:
-      templateName: FL-PRB-GNO-ENG-00133.docx
+      templateName: FL-PRB-GNO-ENG-00133-DTSPB-2638.docx
+    blankLetter:
```

Comment:
bit confused - are we still using 2 templates? I thought you had decided to reuse the existing one in all scenarios?